apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/Proto/Tests.cs

```diff
@@ -908,7 +908,7 @@ public sealed partial class GetTest : pb::IMessage<GetTest> {
       }
       if (other.request_ != null) {
         if (request_ == null) {
```

Comment:
I know these are all generated, but it's a little "weird" to use request_ for the null check and the Request for assigning the value. For these and all the others.

Reply:
Yup, this has changed many times over the course of the protobuf lifetime.